src/CMakeLists.txt

```diff
@@ -52,8 +54,27 @@ elseif(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   set(STDLIB "")
 endif()
 
-if (${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
-   set(STDLIB "${STDLIB} -static-libgcc -static-libstdc++")
+#
+# Identify build type - local or deployment (Travis)
+# the variable NUPIC_DEPLOYMENT_BUILD must be set in travis CI scripts
+#
+if($ENV{NUPIC_DEPLOYMENT_BUILD})
+  message("Doing a deployment build in TravisCI")
+else()
+  message("Doing a local build")
+endif()
+
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
Did you look at the link? It sets these variables in the top level CMakeLists.txt which will apply to things here as well. No need to set it again.